src/firefly/js/charts/ui/options/BasicOptions.jsx

```diff
@@ -377,7 +377,9 @@ export function submitChanges({chartId, fields, tbl_id, renderTreeId}) {
 
         // omit fields, that start with '__'
         if (!k.startsWith('__') && isUndefined(changes[k])) {
-            changes[k] = v;
+            if (layout.title || k!=='layout.title.text' || v) { // if no title set then don't set it in the changes
+                changes[k] = v;
+            }
```

Comment:
I will try, I am not sure it will work though. This situation only happens if `layout.title` is not already set. Also the if only cares about `v`  in the case where the key is `layout.title.text` which means it will be a string.